src/helpers/LatexHelpers.ts

```diff
@@ -0,0 +1,67 @@
+import { CacheType, CommandInteraction, MessageAttachment } from "discord.js";
+import svg2img, { ResvgRenderOptions } from "svg2img";
+var mjAPI = require("mathjax-node");
+
+
+export const Santinize = (message: string): string => {
+    let res_commands = [
+        [`\text{`, `\\backslash text~{`],
+        ['$', `\\$`],
+        ['"', '\\"'],
+        ['\\union','\\cup']
+    ]
+
+    let cleanedMessage = message
+    res_commands.forEach(([searchValue, replaceValue]) => {
+        cleanedMessage = cleanedMessage.replace(searchValue, replaceValue)
+    })
+    return cleanedMessage
+
+}
+
+export const EquationRender = async (cleanedMessage: string, interaction: CommandInteraction<CacheType>) => {
+    mjAPI.config({
+        MathJax: {
+            SVG: {
+                // scale: 70,
+                //font: "STIX-Web",
+            },
+        }
+    });
+    mjAPI.start();
```

Comment:
This may not scale well. I really looked but couldnt find documentation on it. 

if starts the equation engine then it would not scale well because wed need to bring up the engine every time we run this command, it would be wasted resources.

if above and the engine doesnt do anything weird with multiple successive inputs( IE shared state between renders, I dont think its designed this way tho), we should extract it and put it into a `init` function.

if else to either above, we should keep it here.

if we have an init function, we need to have init functions for every module. 

Sorry if this is a bit confusing, im having a difficult time communicating it